options/options.go

```diff
@@ -364,6 +365,12 @@ type TerragruntOptions struct {
 	// StrictControls is a slice of strict controls enabled.
 	StrictControls []string
 
+	// ExperimentMode is a flag to enable experiment mode for terragrunt.
+	ExperimentMode bool
```

Comment:
I think should be updated `Clone()` function too, otherwise can be issues in `run-all` cases

Reply:
We should add a test for this at some point. I always forget to do this.